desktop-app/app/actions/browser.js

```diff
@@ -706,6 +730,50 @@ export function toggleInspector() {
   };
 }
 
+export function toggleCSSEditor() {
```

Comment:
I feel that flipping is more of the `action`'s responsibility.

Here is my understanding. The state in the action here is what is reflected in the UI and what the user is seeing. So I think it makes more sense to let the action decide the final value and send it to the reducer to save it. 
I believe, this way it will be more deterministic. 

But I'll read a bit more about this responsibility splitting and then we can take a call.